classes/Entities/BrandManagerBrand.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Avado\MoodleAbstractionLibrary\Entities;
+
+/**
+ * Class BrandManagerBrand
+ * @package Avado\MoodleAbstractionLibrary\Entities
+ */
+class BrandManagerBrand extends BaseModel
+{
+    /**
+     * @var string
+     */
+    protected $table = 'brandmanager_brand';
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function cohorts()
```

Comment:
can we call this brandCohorts()? There is a cohort table so it would be good to avoid confusion.

Reply:
sure.. Done.